compiler/rustc_codegen_cranelift/src/num.rs

```diff
@@ -368,7 +369,7 @@ pub(crate) fn codegen_ptr_binop<'tcx>(
         .layout()
         .ty
         .builtin_deref(true)
-        .map(|TypeAndMut { ty, mutbl: _ }| !has_ptr_meta(fx.tcx, ty))
+        .map(|RawPtr { ty, mutbl: _ }| !has_ptr_meta(fx.tcx, ty))
```

Comment:
This is misleading. Doing a builtin deref doesn't give you a raw pointer, it gives you the type and mutability of the pointee.